src/codegen/runtime_functions.cpp

```diff
@@ -67,6 +71,34 @@ storage::TileGroup *RuntimeFunctions::GetTileGroup(storage::DataTable *table,
   return tile_group.get();
 }
 
+//===----------------------------------------------------------------------===//
+// Get the Zone Map Manager Instance
+//===----------------------------------------------------------------------===//
+storage::ZoneMapManager *RuntimeFunctions::GetZoneMapManager() {
+  return storage::ZoneMapManager::GetInstance();
+}
+
+//===----------------------------------------------------------------------===//
+// Fills in the Predicate Array for the Zone Map to compare against.
```

Comment:
Add a little more description on what this method is doing.

Reply:
Ive added description in the new commit.